torch/_inductor/compile_fx.py

```diff
@@ -555,24 +556,53 @@ def run(new_inputs):
     return run
 
 
-def count_tangents(fx_g: torch.fx.GraphModule):
-    """
-    Infers which inputs are static for a backwards graph
-    """
+def get_static_input_idxs_for_bw(fx_g: torch.fx.GraphModule):
+    # This function returns the range of ids pointing to the location of parameters in the bwd graph
+    # The input signature for the bwd pass from AOT Autograd is
+    #   (*saved_tensors, *tangents)
+    # When functionalization of rng ops is turned on, we have two more inputs
+    #   (seed, offset, *saved_tensors, *tangents)
```

Comment:
Would it be possible to do `(*saved_tensors, seed, offset, *tangents)` ? That seems a little simpler perhaps ?